src/pipecat/services/openai.py

```diff
@@ -3,45 +3,34 @@
 #
 # SPDX-License-Identifier: BSD 2-Clause License
 #
-
+import base64
 import io
 import json
 import time
-import aiohttp
-import base64
+from typing import AsyncGenerator, List, Literal
 
+import aiohttp
+from loguru import logger
 from PIL import Image
 
-from typing import AsyncGenerator, List, Literal
-
-from pipecat.frames.frames import (
-    ErrorFrame,
-    Frame,
-    LLMFullResponseEndFrame,
-    LLMFullResponseStartFrame,
-    LLMMessagesFrame,
-    LLMResponseEndFrame,
-    LLMResponseStartFrame,
-    TextFrame,
-    URLImageRawFrame,
-    VisionImageRawFrame
-)
-from pipecat.processors.aggregators.openai_llm_context import OpenAILLMContext, OpenAILLMContextFrame
+from pipecat.frames.frames import (AudioRawFrame, ErrorFrame, Frame,
+                                   LLMFullResponseEndFrame,
+                                   LLMFullResponseStartFrame, LLMMessagesFrame,
+                                   LLMResponseEndFrame, LLMResponseStartFrame,
+                                   TextFrame, URLImageRawFrame,
+                                   VisionImageRawFrame)
+from pipecat.processors.aggregators.openai_llm_context import (
+    OpenAILLMContext, OpenAILLMContextFrame)
```

Comment:
Any chance we can keep the current format?

Reply:
In all the imports, I mean. One module per line.

Reply:
Need to configure my imports sorter to do that for me ;)